templates/core/terraform/network/network.tf

```diff
@@ -65,3 +65,39 @@ resource "azurerm_subnet" "resource_processor" {
   # notice that private endpoints do not adhere to NSG rules
   enforce_private_link_endpoint_network_policies = true
 }
+
+resource "azurerm_subnet" "airlock_processor" {
```

Comment:
Where are the NSGs for these subnets?

Reply:
https://github.com/microsoft/AzureTRE/pull/2040/files#diff-af33eb25c46a3f6333e0e1b7d9b0d51306f730f1edbfe659f2fcbe4f5b0bad91R173